easybuild/easyconfigs/o/OpenKIM/OpenKIM-1.8.2-foss-2016b.eb

```diff
@@ -0,0 +1,243 @@
+name = 'OpenKIM'
+version = '1.8.2'
+
+homepage = 'https://openkim.org/'
+description = """Open Knowledgebase of Interatomic Models.
+
+OpenKIM is an API and a collection of interatomic models (potentials) for
+atomistic simulations.  It is a library that can be used by simulation programs
+to get access to the models in the OpenKIM database.
+"""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = ['kim-api-v%(version)s.tgz']
+
+source_urls = [
+    'https://openkim.org/download/',    # For models and drivers
+    'https://s3.openkim.org/kim-api/',  # For API files
+]
+
+parallel = 1
+
+modextrapaths = {'KIM_HOME': '.'}
+
+moduleclass = 'chem'
+
+# Sources for all KIM items.  Autogenerated by make-kim-model-list.py
+# AUTOGENERATED ON: Fri Jun 30 08:44:55 2017 UTC
+sources += [
```

Comment:
@schiotz OK to include like this, but do move it up below the `sources =` line

At some point, we will start for checking the order of easyconfig parameters as well in the automated style check